jetson/nucleo_bridge/Controller.cpp

```diff
@@ -102,7 +102,7 @@ void Controller::refresh_calibration_data()
 
         transact(CALIBRATED, nullptr, UINT8_POINTER_T(&raw_calibration_data));
 
-        calibrated = raw_calibration_data == 0xF;
+        calibrated = raw_calibration_data != 0;

```

Comment:
can this be a constant or something? why !=0? plz put in comment

Reply:
Done kinda. raw_calibration_data is either 0xFF or 0x00 where 0 means not calibrated and 0xFF means calibrated